venues/aclweb.org/ACL/post_assignment_edges.py

```diff
@@ -41,4 +40,4 @@ def assign_reviewer_to_paper(reviewer_id_or_email, paper_forum):
         signatures=["aclweb.org/ACL/2022/Conference/Ethics_Chairs"]
     ))
 
-assign_reviewer_to_paper('~Melisa_Bok1','Ofoi1kAeB5o')
\ No newline at end of file
+assign_reviewer_to_paper("","")
```

Comment:
you can set the forum and reviewer as parameter of the script:

parser.add_argument('-a','--add',help="username or email address to add")
parser.add_argument('-r','--remove',help="username or email address to remove")
parser.add_argument('-n','--number',help="the paper number")

We can ignore the remove parameter for now